public/css/themes/solarized-dark.less

```diff
@@ -266,3 +317,65 @@ textarea {
     color: @icinga-blue;
   }
 }
+
+/* Form styles */
+
+@input-background: lighten(@base02, 5);
+
+.icinga-form,
+.controls {
+  input[type="text"],
+  input[type="password"],
+  input[type="number"],
+  input[type="datetime-local"],
+  input[type="date"],
+  input[type="time"],
+  textarea,
+  select {
+    background-color: @input-background;
+  }
+
+  & .toggle-switch .toggle-slider {
+    background: @input-background;
+    border: 2px solid @input-background;
+  }
+
+  & .toggle-switch .toggle-slider:before {
+    background: @body-bg-color;
+    border: 1px solid @body-bg-color;
+  }
+
+  & input[type="checkbox"]:checked + .toggle-switch .toggle-slider {
+    background-color: @button-primary-color;
+    border: 1px solid @button-primary-color;
+  }
+
+  & input[type="checkbox"]:focus + .toggle-switch .toggle-slider {
+    box-shadow: 0 0 0 2px @body-bg-color, 0 0 0 4px fade(@button-primary-color, 40);
+  }
+
+  & input[type="checkbox"]:checked + .toggle-switch .toggle-slider:before {
+    border: 1px solid @button-primary-color;
+  }
+}
+
+.button {
```

Comment:
Does this collide with the mixin?

Reply:
It doesn't, it just overwrites the colours used